movement_primitives/dmp/_dmp.py

```diff
@@ -255,10 +255,10 @@ def dmp_step_euler(
     start_t : float
         Time at the start.
 
-    alpha_y : float
+    alpha_y : array, shape (n_dims,)
```

Comment:
* It seems like float is still a valid option and will be "broadcasted". This should be documented.
* The default value should also be documented.